pom.xml

```diff
@@ -33,6 +33,7 @@
     <stack.version>5.0.0-SNAPSHOT</stack.version>
     <doc.skip>true</doc.skip>
     <jar.manifest>${project.basedir}/src/main/resources/META-INF/MANIFEST.MF</jar.manifest>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
Shouldn't we consider bumping to a 17 baseline?

Reply:
we stick to 11 for Vert.x 5.0